src/Core/Domain/CartRule/ValueObject/PercentageDiscount.php

```diff
@@ -33,6 +33,8 @@
  */
 class PercentageDiscount
 {
+    const MAX_PERCENTAGE = 100;
```

Comment:
Why don't add a MIN_PERCENTAGE ?

Reply:
what would it be? Min percentage should be just greater than zero, but not the zero itself. So would it be good to make MIN_PERCENTAGE = 0.000001 ?